aim/storage/union.py

```diff
@@ -205,6 +205,7 @@ def get(self, key: bytes, *args, **kwargs) -> bytes:
             # Shadowing
             if key.startswith(prefix):
                 return db.get(key)
+        return self.dbs[b""].get(key)
         raise ValueError
```

Comment:
this line should be removed

Reply:
I don't know why byt Github shows the comment for line 208. What I mean, is that line `209` should be removed.